### ISSUE TITLE
No Support for nodeFilterEditable

### ISSUE THREAD

**Issue by dgarstang** [rundeck/terraform-provider-rundeck#90]
It seems that the rundeck_job provider doesn't support the ability to make a job's node filter not editable. In the yaml representation of a job, this is the `nodeFilterEditable` key at the top level of a rundeck job.

### Terraform Version
Terraform v 1.2.3

### Affected Resource(s)
Please list the resources as a list, for example:
- rundeck_job

**Comment by fdevans** [comment]
This is correct.  As this is a community supported project we are open to Pull Requests that would add this as a option.
